src/scene/assetmanager_lua.inl

```diff
@@ -41,6 +41,13 @@ namespace {
     openspace::global::openSpaceEngine->assetManager().remove(assetName);
 }
 
+/**
+ * Reloads (remove+add) the asset with the specfied name.
```

Comment:
```suggestion
 * Reloads the asset with the specified name. If the asset was previously loaded explicity
 * it will be removed and then re-added. If the asset was not previously loaded, it will only
 * be loaded instead.
```